lib/SILOptimizer/SILCombiner/SILCombinerMiscVisitors.cpp

```diff
@@ -465,6 +465,34 @@ SILInstruction *SILCombiner::visitAllocRefInst(AllocRefInst *AR) {
   return nullptr;
 }
 
+SILInstruction *SILCombiner::visitReallocRefInst(ReallocRefInst *AR) {
```

Comment:
This is an optimization. If we want to do this, lets do it later, in a different PR.